test/unit/carriers/ups_test.rb

```diff
@@ -603,4 +603,13 @@ def test_package_surepost_less_than_one_lb_service_code
     assert_equal 'OZS', request.search('/Package/PackageWeight/UnitOfMeasurement/Code').text
     assert_equal '8.0', request.search('/Package/PackageWeight/Weight').text
   end
+
+  def test_address_validation
+    location = Location.new(address1: "55 Glenlake Parkway", city: "Atlanta", state: "GA", zip: "30328", country: "US")
+    address_validation_response = xml_fixture('ups/address_validation_response')
+    @carrier.expects(:commit).returns(address_validation_response)
+    response = @carrier.validate_address(location)
+    assert_equal :commercial, response.classification
```

Comment:
An added check for the `validity` seems like a good idea here. I mentioned it above, but I'd like to see some tests checking different validity responses, since right now we're only verifying the sunny day scenario.